1.0/18-alpine3.17/docker-entrypoint.sh

```diff
@@ -9,7 +9,7 @@ fi
 
 function wait_tcp_port {
     local host="$1" port="$2"
-    local max_tries=5 tries=1
+    local max_tries="$3" tries=1
```

Comment:
@rtritto when I look at line 32, there are only two arguments, so max_tries will always be set to ""? Unless I'm misunderstanding?

Reply:
`docker-entrypoint.sh` are correctly regenerated in #101 using `docker-entrypoint.sh` at root.